grpc/src/main/java/com/linecorp/armeria/server/grpc/HttpJsonTranscodingPathParser.java

```diff
@@ -484,12 +482,26 @@ static String asParameterizedPath(List<PathSegment> segments, boolean withLeadin
          */
         static String asGlobPath(List<PathSegment> segments, boolean withLeadingSlash) {
             requireNonNull(segments, "segments");
-            final String path = segments.stream()
-                                        .map(segment -> segment.segmentString(PathMappingType.GLOB))
-                                        .collect(Collectors.joining("/"));
+            final String path = toPathString(segments, PathMappingType.GLOB);
             return withLeadingSlash ? '/' + path : path;
         }
 
+        private static String toPathString(List<PathSegment> segments, PathMappingType type) {
+            final PathSegment lastSegment = segments.get(segments.size() - 1);
+            if (lastSegment instanceof VerbPathSegment) {
```

Comment:
Question: Shouldn't we do some validation here such as checking if the body `"*"` when POST is used? 🤔 
https://github.com/line/armeria/blob/17b64b976dc8fee4de6757cd8836053faf859f0c/grpc/src/main/java/com/linecorp/armeria/server/grpc/HttpJsonTranscodingPathParser.java#L67
https://cloud.google.com/apis/design/custom_methods#http_mapping

Reply:
Thank you for your suggestion! As you said, I think we need to validate the verb part so I used a regex to make sure the verb part is in valid. 
In my opinion, we don't need to validate if the body is `"*"` when POST is used because some poeple might want to use a body other than `*` according to this [stackoverflow](https://stackoverflow.com/questions/60029341/why-shouldnt-custom-methods-use-the-url-for-transferring-data). What do you think of this?